content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -189,6 +189,15 @@ spec:
 
 This ensures that even pods that aren't selected by any other NetworkPolicy will not be allowed ingress or egress traffic.
 
+## SCTP support
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+Kubernetes introduces the support of SCTP as a `protocol` value in `NetworkPolicy` definitions as an alpha feature. To enable this feature, the cluster administrator needs to enable the `SCTPSupport` feature gate on the apiserver, for example, `“--feature-gates=SCTPSupport=true,...”`. When the feature gate is enabled, users can set the `protocol` field of a `NetworkPolicy` to `SCTP`. Kubernetes sets up the network accordingly for the SCTP associations, just like it does for TCP connections.
```

Comment:
I don't know if we want words like "introduces" which are anchored in time, vs something like "supports". 

I guess that, since this is related to the feature-gate, it might be OK.

@chenopis

Reply:
OK, I change it to "supports".